.golangci.yml

```diff
@@ -1,5 +1,5 @@
-service:
-  project-path: github.com/codeready-toolchain/toolchain-e2e
+#service:
```

Comment:
let's just remove these lines, instead of keeping them commented out

Reply:
yes we can.. i just didnt know why we had these lines in the first place so just commented it to check and get review

Reply:
This is probably some legacy/outdated config and as you pointed out, since v6.5.0, golangci-lint action validates the config and starts failing. We simply didn't realize these lines had been invalid for a long time 🤷‍♂️